README.md

```diff
@@ -78,6 +78,21 @@ source ~/.bashrc
 ```
 > ***Note:*** The ROS2 daemon may need to be restarted to use the CycloneDDS RMW. Use `ros2 daemon stop` and `ros2 daemon start` to restart the daemon.
 
+### ROS 2 navigation stack
+Install the required ROS2 navigation dependencies:
+```bash
+sudo apt install ros-${ROS_DISTRO}-navigation2
```

Comment:
There is a whole section about required packages later in the text. It would be helpful for the user to keep all requirements in one spot, making it more error-prone (it is easier to skip one command if there are more commands).

Moreover `ros-${ROS_DISTRO}-navigation2` is a meta-package installing all possible navigation packages. It would be good to either install this one, or pick only necessary packages. What I do not like is that it depends on other packages, such as `ros-${ROS_DISTRO}-nav2-msgs` listed later on. We do not want to list packages twice.